manifests/u/Ubisoft/Connect/124.2.0.10565/Ubisoft.Connect.installer.yaml

```
@@ -0,0 +1,20 @@
+# Automatically updated by the winget bot at 2021/Jul/30
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Ubisoft.Connect
+PackageVersion: 124.2.0.10565
+Platfom:

```

```suggestion
Platform:
```
cc @OfficialEsco: you made a typo here.

---

Gotta love those quick manual edits after validation 🙄